packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -40,7 +40,7 @@ const propTypes = {
   /**
    * intl object programmatically imported through injectIntl from react-intl.
    * */
-  intl: intlShape.isRequired,
+  intl: PropTypes.shape({ locale: PropTypes.string }).isRequired,
```

Comment:
I know it won't happen now, but do you think we should utilize react-intl's timeZone config option in the intlProvider?

Reply:
I don't see why not. We could easily add that as a prop to ApplicationBase to allow that to be set.